### PR TITLE
PPS-163: Update Prerender service documentation

### PR DESCRIPTION
**Pull request by sandro-macrometa**
Jira: https://macrometa.atlassian.net/browse/PPS-163

- Small fix in the interactions page
- Open API spec with updated response examples. The attributes `status`, `message`, and `data` were removed from the response. Now the `data` is returned directly

### CODE REVIEW

File: static/openapi/prerendering-spec.json [hunk 1]
```diff
@@ -302,89 +250,71 @@
...
+                        "format": "uri",
+                        "description": "URL of the content that has been cached."
+                      },
+                      "statusCode": {
+                        "type": "number",
+                        "description": "Response HTTP status code."
+                      },
+                      "isPrefetch": {
+                        "type": "boolean",
+                        "description": "Whether rendering was performed as a prefetch operation (true) or on-demand (false)."
```
Comment (diana-macrometa) [inline static/openapi/prerendering-spec.json]:
```suggestion
                        "description": "Whether rendering was performed as a prefetch operation (true) or on demand (false)."
```

File: static/openapi/prerendering-spec.json [hunk 2]
```diff
@@ -5910,12 +5581,38 @@
...
+        "summary": "Get prerender service telemetry",
+        "tags": ["Telemetry"],
+        "description": "Returns prerender service telemetry.",
+        "parameters": [
+          {
+            "schema": { "type": "string" },
+            "in": "header",
+            "name": "x-photoniq-customerid",
+            "required": true,
+            "description": "The unique identifier for the customer"
```
Comment (diana-macrometa) [inline static/openapi/prerendering-spec.json]:
```suggestion
            "description": "The unique identifier for the customer."
```

File: static/openapi/prerendering-spec.json [hunk 3]
```diff
@@ -5910,12 +5581,38 @@
...
           },
+          {
+            "schema": { "type": "string" },
+            "in": "header",
+            "name": "Authorization",
+            "required": true,
+            "description": "Authorization token for API access"
```
Comment (diana-macrometa) [inline static/openapi/prerendering-spec.json]:
```suggestion
            "description": "Authorization token for API access."
```

**Review by diana-macrometa — COMMENTED** [review]
LGTM, I would like @joubert-macrometa to take a look before this is merged.

File: static/openapi/prerendering-spec.json (hunk 3 above)
Comment (sandro-macrometa) [reply static/openapi/prerendering-spec.json]: Done!

File: static/openapi/prerendering-spec.json (hunk 2 above)
Comment (sandro-macrometa) [reply static/openapi/prerendering-spec.json]: Done!

File: static/openapi/prerendering-spec.json (hunk 1 above)
Comment (sandro-macrometa) [reply static/openapi/prerendering-spec.json]: Done!

**Comment by diana-macrometa** [comment]
@joubert-macrometa, please review.

**Review by joubert-macrometa — APPROVED** [review]

**Pull request merged**